booking/src/test/java/DBUtilTest.java

```diff
@@ -0,0 +1,122 @@
+        try {
+            ps = conn.prepareStatement("insert into accomodation" +
+                    " (id, type, bed_type, max_guests, description) values (?, ?, ?, ?, ?)");
+            ps2 = conn.prepareStatement("insert into room_fair (id, value, season) values (?, ?, ?)");
+            ps3 = conn.prepareStatement("insert into accomodation_fair_relation " +
+                    "(id, id_accomodation, id_room_fair) values (?, ?, ?)");
+            int step = 0;
+            for (int i = 0; i < accomodations.size(); i++) {
+                ps.setInt(1, accomodations.get(i).getId());
+                ps.setString(2, accomodations.get(i).getType());
+                ps.setString(3, accomodations.get(i).getBed_type());
+                ps.setInt(4, accomodations.get(i).getMax_guests());
+                ps.setString(5, accomodations.get(i).getDescription());
+                ps2.setInt(1, roomFairs.get(i + step).getId());
+                ps2.setDouble(2, roomFairs.get(i + step).getValue());
+                ps2.setString(3, roomFairs.get(i + step).getSeason());
+                ps3.setInt(1, i + step);
+                ps3.setInt(2, accomodations.get(i).getId());
+                ps3.setInt(3, roomFairs.get(i + step).getId());
+                ps.executeUpdate();
+                ps2.executeUpdate();
+                ps3.executeUpdate();
+                //step is used to insert two values in room_fair coresponding to the fair for the room in accomodation
+                //for each season
+                step++;
+                ps2.setInt(1, roomFairs.get(i + step).getId());
+                ps2.setDouble(2, roomFairs.get(i + step).getValue());
+                ps2.setString(3, roomFairs.get(i + step).getSeason());
+                ps3.setInt(1, i + step);
+                ps3.setInt(2, accomodations.get(i).getId());
+                ps3.setInt(3, roomFairs.get(i + step).getId());
+                ps2.executeUpdate();
+                ps3.executeUpdate();
+            }
+        } catch (SQLException e) {
+            System.err.println("Cannot insert database: " + e.getMessage());
+        } finally {
+            if (ps != null) try {
+                ps.close();
+            } catch (SQLException e) {
+            }
+        }
+    }
+
+    /**
+     * @throws SQLException Collects data from the DB and prints it out to the console
+     *                      Uses DBUtil, BookingDao and Booking
+     */
+    @org.junit.Test
+    public void extractFromDB() throws SQLException {
+        DBUtil dbUtil = new DBUtil();
+        Connection conn = dbUtil.connection("booking");
+        if (conn == null)
+            return;
+        BookingDAO bookingDAO = new BookingDAO(conn);
+        List<Booking> bookingTypes = bookingDAO.getAllAsList();
+        for (Booking b : bookingTypes) {
+            System.out.println("Room type:" + b.getType() + " -- price: " +
+                    b.getValue().toString() + " -- during: " + b.getSeason());
```

Comment:
i would have expected to see an assertion here. In unit tests we usually test that what we get is what we expect